openpathsampling/pathmover.py

```diff
@@ -1616,17 +1616,14 @@ class MinusMover(WrappedMover):
     paths between the innermost regular TIS interface ensemble and the minus
     interface ensemble. This is particularly useful for improving sampling
     of path space.
-
-    Note that the inheritance from ReplicaExchangeMover is only to assist
-    with `isinstance` in later anealysis. Since the only two functions here
-    are `.__init__() and `.move()`, both of which exist in both parent
-    classes, the calls to `super` will use the version in
-    ConditionalSequentalMover. However, analysis routines will see
-    `isinstance(minus, ReplicaExchangeMover)` as True.
     """
-    def __init__(self, minus_ensemble, innermost_ensemble, 
-                 ensembles=None):
+    def __init__(self, minus_ensemble, innermost_ensembles, ensembles=None):
         segment = minus_ensemble._segment_ensemble
+        try:
+            innermost_ensembles = list(innermost_ensembles)
```

Comment:
:-) I've been trying to find a good way to "clean" the input so that it is a list. This is the version I like the most. As I come across others, I think I'll change them so that they are like this.